### ISSUE TITLE
[BUG] Error while saving entries with neo fields

### ISSUE THREAD

**Issue by mihob** [ryssbowh/craft-activity#13]
The following error occours while saving entries with neo fields:

[yii\base\UnknownPropertyException] Getting unknown property: craft\fieldlayoutelements\HorizontalRule::field

```
{
    "trace":[
        "#0 /app/vendor/ryssbowh/craft-activity/src/models/fieldHandlers/elements/Neo.php(178): yii\\base\\Component->__get('field')",
        "#1 /app/vendor/ryssbowh/craft-activity/src/models/fieldHandlers/elements/Neo.php(159): Ryssbowh\\Activity\\models\\fieldHandlers\\elements\\Neo->buildBlockValues(Object(benf\\neo\\elements\\Block))",
        "#2 /app/vendor/ryssbowh/craft-activity/src/models/fieldHandlers/elements/Neo.php(27): Ryssbowh\\Activity\\models\\fieldHandlers\\elements\\Neo->buildValues()",
        "#3 /app/vendor/yiisoft/yii2/base/BaseObject.php(109): Ryssbowh\\Activity\\models\\fieldHandlers\\elements\\Neo->init()",
        "#4 /app/vendor/craftcms/cms/src/base/Model.php(78): yii\\base\\BaseObject->__construct(Array)",
        "#5 /app/vendor/ryssbowh/craft-activity/src/traits/ElementFields.php(65): craft\\base\\Model->__construct(Array)",
        "#6 /app/vendor/ryssbowh/craft-activity/src/recorders/Entries.php(116): Ryssbowh\\Activity\\base\\recorders\\ElementsRecorder->getCustomFieldValues(Object(craft\\elements\\Entry))",
        "#7 /app/vendor/ryssbowh/craft-activity/src/base/recorders/ElementsRecorder.php(34): Ryssbowh\\Activity\\recorders\\Entries->getFieldsValues(Object(craft\\elements\\Entry))",
        "#8 /app/vendor/ryssbowh/craft-activity/src/recorders/Entries.php(49): Ryssbowh\\Activity\\base\\recorders\\ElementsRecorder->beforeSaved(Object(craft\\elements\\Entry))",
        "#9 [internal function]: Ryssbowh\\Activity\\recorders\\Entries->Ryssbowh\\Activity\\recorders\\{closure}(Object(craft\\events\\ModelEvent))",
        "#10 /app/vendor/yiisoft/yii2/base/Event.php(312): call_user_func(Object(Closure), Object(craft\\events\\ModelEvent))",
        "#11 /app/vendor/yiisoft/yii2/base/Component.php(642): yii\\base\\Event::trigger('craft\\\\elements\\\\...', 'beforeSave', Object(craft\\events\\ModelEvent))",
        "#12 /app/vendor/craftcms/cms/src/base/Element.php(4917): yii\\base\\Component->trigger('beforeSave', Object(craft\\events\\ModelEvent))",
        "#13 /app/vendor/craftcms/cms/src/elements/Entry.php(1881): craft\\base\\Element->beforeSave(false)",
        "#14 /app/vendor/craftcms/cms/src/services/Elements.php(2931): craft\\elements\\Entry->beforeSave(false)",
        "#15 /app/vendor/craftcms/cms/src/services/Elements.php(1535): craft\\services\\Elements->_saveElementInternal(Object(craft\\elements\\Entry), false, false, NULL, Array)",
        "#16 /app/vendor/craftcms/cms/src/services/Elements.php(1180): craft\\services\\Elements->duplicateElement(Object(craft\\elements\\Entry), Array)",
        "#17 /app/vendor/craftcms/cms/src/services/Drafts.php(304): craft\\services\\Elements->updateCanonicalElement(Object(craft\\elements\\Entry), Array)",
        "#18 /app/vendor/craftcms/cms/src/controllers/ElementsController.php(1352): craft\\services\\Drafts->applyDraft(Object(craft\\elements\\Entry))",
        "#19 [internal function]: craft\\controllers\\ElementsController->actionApplyDraft()",
        "#20 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)",
        "#21 /app/vendor/yiisoft/yii2/base/Controller.php(178): yii\\base\\InlineAction->runWithParams(Array)",
        "#22 /app/vendor/yiisoft/yii2/base/Module.php(552): yii\\base\\Controller->runAction('apply-draft', Array)",
        "#23 /app/vendor/craftcms/cms/src/web/Application.php(302): yii\\base\\Module->runAction('elements/apply-...', Array)",
        "#24 /app/vendor/craftcms/cms/src/web/Application.php(627): craft\\web\\Application->runAction('elements/apply-...', Array)",
        "#25 /app/vendor/craftcms/cms/src/web/Application.php(281): craft\\web\\Application->_processActionRequest(Object(craft\\web\\Request))",
        "#26 /app/vendor/yiisoft/yii2/base/Application.php(384): craft\\web\\Application->handleRequest(Object(craft\\web\\Request))",
        "#27 /app/web/index.php(27): yii\\base\\Application->run()",
        "#28 {main}"
        ],
    "memory":20944240,
    "exception":"[object] (yii\\base\\UnknownPropertyException(code: 0): Getting unknown property: craft\\fieldlayoutelements\\HorizontalRule::field at /app/vendor/yiisoft/yii2/base/Component.php:154)"
}
```

Craft Version: 4.4.5
Plugin Version: 2.3.7

**Comment by ryssbowh** [comment]
Thanks for reporting,

This has been fixed in 2.3.8

**Comment by mihob** [comment]
Thx! It's working in the actual version
